hopsworks-common/src/main/java/io/hops/hopsworks/common/dao/jupyter/config/JupyterProcessMgr.java

```diff
@@ -275,16 +275,29 @@ public void killServerJupyterUser(String hdfsUsername, String jupyterHomePath, L
 
   }
 
+  public void stopCleanly(String hdfsUser, int port) throws ServiceException {
+    // We need to stop the jupyter notebook server with the PID
+    // If we can't stop the server, delete the Entity bean anyway
```

Comment:
@robzor92 in this case, the JupyterCleaner timer will clean it eventually?

Reply:
I removed this method, it is not needed anymore. I will use another one for the cleanup. Right now we have some duplicated cleanup code in the JupyterCleaner and JupyterService for shutting down the notebook server.